### PR TITLE
code challenge

### PR DESCRIPTION
**Pull request by beccaelenzil**

### CODE REVIEW

**Review by CheezItMan — APPROVED** [review]
Nice work!

File: 07/reshape-a-matrix.checkpoint.md [hunk 1]
```diff
@@ -198,26 +198,72 @@ Steps to solve a subproblem
...
+    def test_example_2(self):
+        # Arrange
+        matrix = [[1,2],[3,4]]
+        r = 2
+        c = 4
+        # Act
+        answer = p.reshape_matrix(matrix, r, c)
+        # Assert
+        self.assertEqual([[1,2],[3,4]], answer) 
+    def test_example_3(self):
```
Comment (anselrognlie) [inline 07/reshape-a-matrix.checkpoint.md:252]: Would it be worth adding  a description for this test case to the explanation?

**Review by anselrognlie — COMMENTED** [review]
I noticed that two of the tests are discussed in the explanation, but not the third. Is it worth adding that to the problem prompt?

File: 07/reshape-a-matrix.checkpoint.md (hunk 1 above)
Comment (beccaelenzil) [reply 07/reshape-a-matrix.checkpoint.md:252]: Yes, I'll do this!

**Review by anselrognlie — APPROVED** [review]
👍

**Pull request merged**